### PR TITLE
Add project gql fetch by coinmarketcap

### PR DESCRIPTION
**Pull request by IvanIvanoff**

### CODE REVIEW

File: lib/sanbase_web/graphql/resolvers/project_resolver.ex [hunk 1]
```diff
@@ -55,6 +53,23 @@ defmodule SanbaseWeb.Graphql.Resolvers.ProjectResolver do
...
+      Project
+      |> Repo.get_by(coinmarketcap_id: slug)
+      |> case do
+        nil ->
+          {:error, "Project with slug '#{slug}' not found."}
+
+        project ->
+          project =
+            project
+            |> Repo.preload([:latest_coinmarketcap_data, icos: [ico_currencies: [:currency]]])
```
Comment (valo) [inline lib/sanbase_web/graphql/resolvers/project_resolver.ex]: It will be so nice to remove these preloads, but I guess this is outside of the scope of this PR

Comment (IvanIvanoff) [reply lib/sanbase_web/graphql/resolvers/project_resolver.ex]: It's exactly like the project fetching by id. Should be rewritten to use the dataloader in another PR
